robot-db/src/main/java/domain/Books.java

```diff
@@ -1,32 +0,0 @@
-package domain;
-
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-
-/**
- * First idea of entity representing books.
```

Comment:
I know it's deleted code, but this is actually fairly interesting piece of info in a Javadoc. First idea - nice. :-)